pkg/cmd/deployment/root.go

```diff
@@ -71,9 +72,12 @@ var deploymentApplyCmd = &cobra.Command{
 }
 
 var deploymentDeleteCmd = &cobra.Command{
-	Use:   "delete [handlerGlob]",
+	Use:   "delete",
 	Short: "Delete an application deployment",
 	Long:  `Delete a Nitric application deployment.`,
+	Example: `nitric delete -n prod-aws
```

Comment:
Is there a alias/shortcut setup for this or should this be `nitric deployment delete`

Reply:
the deployment name defaults to "dep", so it's not required